core/services/relay/evm/chain_writer.go

```diff
@@ -186,7 +186,7 @@ func (w *chainWriter) GetFeeComponents(ctx context.Context) (*commontypes.ChainF
 		return nil, fmt.Errorf("gas estimator not available")
 	}
 
-	fee, _, err := w.ge.GetFee(ctx, nil, 0, w.maxGasPrice)
+	fee, _, err := w.ge.GetFee(ctx, nil, 0, w.maxGasPrice, nil)
```

Comment:
If `e.geCfg.EstimateGasLimit()` is enabled, will we get/log error in this case with empty callMsg?

Reply:
I don't think we would because all of the parameters for the `eth_estimateGas` call are actually optional. We'd probably get some default value if `EstimateGasLimit` is enabled but it won't be used in cases like this.

Reply:
Joe does have a point though that if CW wants to access gas limits they would have to update their API and provide a `to` address. Something to keep in mind.

Reply:
Right, they would. If we move forward with exposing EstimateGas to products, we had discussed we'd do it through this. But they'd have to update the other parameters too.